server/e2e/seeder.go

```
@@ -26,13 +25,7 @@
 	pName  = "p1"
 	pAlias = "PROJECT_ALIAS"
 
-	sID    = id.NewSceneID()
-	dssID  = id.NewDatasetSchemaID()
-	dsID   = id.NewDatasetID()
-	dsfID1 = dataset.NewFieldID()
-	dsfID2 = dataset.NewFieldID()
-	dsfID3 = dataset.NewFieldID()
-	dsfID4 = dataset.NewFieldID()
+	sID = id.NewSceneID()
```

golangci-lint (1.62.2)

var `sID` is unused

(unused)